backend/research_groups/views.py

```diff
@@ -108,18 +119,20 @@ class ResearchGroupViewSet(PermissionPolicyMixin, viewsets.ModelViewSet):
     }
 
     def create(self, request, *args, **kwargs):
-        # Obecnie, w przypadku gdy nie ma użytkownika z podanym mailem, tworzony jest
-        # nowy ale później można tu zaimplementować rozsyłanie maili
         member_emails = request.data["members"]
         member_emails.append(self.request.user.email)
 
+        new_users = []
+
         for member_email in member_emails:
             if not User.objects.filter(email=member_email).exists():
-                User.objects.create_user(
-                    username=member_email.split("@")[0],
-                    password=member_email.split("@")[0],
-                    email=member_email,
+                randUsername = "".join(random.choice(string.ascii_letters) for x in range(150))
+                randPass = "".join(random.choice(string.ascii_letters) for x in range(150))
```

Comment:
To nie będzie za długie?

Reply:
jest na styk

Reply:
W sensie czy to że dostanie taki długi username i hasło to nie za dużo?

Reply:
Testowałem to tak jak jest tu zaimplementowane i nie było problemu.
Chodzi o to, że jak ktoś tworzy koło i podaje emaile członków, którzy nie mają konta, to musiałem jakoś stworzyć userów z tymi mailami, a login i hasło są required, więc robie losowe stringi